internal/runtime/runtime.go

```diff
@@ -38,11 +40,12 @@ const unmarshalErrorMessage = "Unable to unmarshal message payload as %s"
 
 // GolangRuntime represents the golang runtime environment
 type GolangRuntime struct {
-	TargetType    interface{}
-	transforms    []appcontext.AppFunction
-	isBusyCopying sync.Mutex
-	storeForward  storeForwardInfo
-	ServiceKey    string
+	TargetType     interface{}
+	transforms     []appcontext.AppFunction
+	isBusyCopying  sync.Mutex
+	storeForward   storeForwardInfo
+	ServiceKey     string
+	secretProvider *security.SecretProvider
```

Comment:
Just my opinion, put the public fields (those exported) first, and internal / private ones in next.